lib/components/Button/Button.js

```diff
@@ -54,15 +54,17 @@ const Button = forwardRef(({ variant, size, fullWidth, danger, children, ...butt
 });
 
 Button.propTypes = {
-  variant: PropTypes.oneOf(Object.values(VARIANT)).isRequired,
-  size: PropTypes.oneOf(Object.values(SIZE)).isRequired,
+  variant: PropTypes.oneOf(Object.values(VARIANT)),
+  size: PropTypes.oneOf(Object.values(SIZE)),
   type: PropTypes.oneOf(['button', 'submit', 'reset']),
   fullWidth: PropTypes.bool,
   children: PropTypes.node,
   danger: PropTypes.bool,
 };
 
 Button.defaultProps = {
+  variant: VARIANT.PRIMARY,
```

Comment:
@NathanAB I've switched this to primary since this was your preference and now you represent the Core team.

Reply:
Additionally, it makes sense to me that the default === primary. Otherwise, seems like we should switch the colors.

Reply:
hahah ok, thanks